### DISCUSSION TITLE
Places counter is arbitrary?

### DISCUSSION THREAD

**Discussion by heitorPB** [photoprism/photoprism#3874] [Q&A]
#### 1. What is not working as documented?

I added 3 folders of files into my library, all files properly geotagged, around 10 files per folder:
- one from my city
- one from a trip to another state (2 cities)
- one with photos from 2 cities in another continent

This is what I got in the Places "tab":

![places](https://github.com/photoprism/photoprism/assets/13461702/9cf4ef06-4086-4c28-bf77-add58f7a20d7)

I expected the counter for Places to be 5: one for each city of the photos. But instead I got 9.
I couldn't find anything in the docs about what the items in the Places drop-down mean.

Also, I thought `Countries 2` to show up there.

#### 2. How can we reproduce it?

- add photos (with the geotags) to Originals
- import them
- go to places

#### 3. What behavior do you expect?

More (and more meaningful) information in that tab/menu.

#### 4. What could be the cause of your problem?

Maybe the name of the menu item is not as descriptive? Maybe rename `Places` -> `Map`?

Maybe identify/show Countries, States and Cities? With a counter for each one? It currently only shows States for me (and they are correctly identified! I didn't expect the states to appear there, really nice to see it!). Clicking in the States shows both states as albums, this is really awesome and I'd like to see the same for Cities/Countries.

Maybe this is not exactly a bug, but a bug + (feature + doc) request?

#### 5. Can you provide us with example files for testing, error logs, or screenshots?

I can send some files for testing if you need, but the ones I used are private.

#### 6. Which software versions do you use?

(a) PhotoPrism Architecture & Build Number: PhotoPrism® CE Build 231021-9abea5b55

(b) Database Type & Version: MariaDB

(c) Operating System Types & Versions: Linux

(d) Browser Types & Versions: Firefox

(e) Ad Blockers, Browser Plugins, and/or Firewall Software? Nothing that would change the geotags?

#### 7. On what kind of device is PhotoPrism installed?

(a) Device / Processor Type: Laptop, Linux, i9-11900H

(b) Physical Memory & Swap Space: 32G & 32G

(c) Storage Type: NVMe

(d) Anything else that might be helpful to know? just spinned up photoprism from the docker-compose in the docs.

#### 8. Do you use a Reverse Proxy, Firewall, VPN, or CDN?

No.

**Comment by lastzero** [comment]
It's counting distinct location labels:

https://github.com/photoprism/photoprism/blob/1b41e89cbd72cfad2fa0b092db93734de6e7a424/internal/entity/place.go#L17-L22

Also note the second count is *States* (as in United States), not *Countries*.

**Reply by lastzero** [reply]
@heitorPB You are welcome to help improve our documentation so that these details are mentioned where you would expect them:
- https://github.com/photoprism/photoprism-docs

**Reply by heitorPB** [reply]
Oh, cool!

> Also note the second count is States (as in United States), not Countries.

It counted two distinct states here in Brazil: Minas Gerais and São Paulo :smile:

I wonder why it didn't detect the city and country in any of my photos.

> You are welcome to help improve our documentation

As soon as I understand this better!
